packages/pangea-sdk/PangeaCyber.Net/AttachedFile.cs

```diff
@@ -0,0 +1,63 @@
+﻿using PangeaCyber.Net.Exceptions;
+
+namespace PangeaCyber.Net
+{
+    ///
```

Comment:
I'd like to see public APIs be documented more often going forward. Or at the very least, remove these empty `///`s everywhere because they kinda give off the impression: "hey, we were _gonna_ write a doc comment here, but didn't."